flang/lib/Lower/Allocatable.cpp

```diff
@@ -531,11 +543,12 @@ void Fortran::lower::genDeallocateStmt(
                statOrErr.u);
   ErrorManager errorManager;
   errorManager.init(converter, loc, statExpr, errMsgExpr);
-  auto &builder = converter.getFirOpBuilder();
-  auto insertPt = builder.saveInsertionPoint();
-  for (const auto &allocateObject :
+  fir::FirOpBuilder &builder = converter.getFirOpBuilder();
+  mlir::OpBuilder::InsertPoint insertPt = builder.saveInsertionPoint();
+  for (const Fortran::parser::AllocateObject &allocateObject :
        std::get<std::list<Fortran::parser::AllocateObject>>(stmt.t)) {
```

Comment:
This block seems to echo types on these 3 lines to me.

Reply:
If I wasn't clear, it reads:

builder builder get builder;
insertion point insertion point save insertion point;
fortran parser allocate object allocate object get list fortran parser allocate object;